docs/start/tutorial.md

```diff
@@ -282,12 +282,14 @@ import {
 
 // existing imports & code
 
-export default function Root() {
+export default function App() {
   return (
     <html lang="en">
       {/* other elements */}
       <body>
+        <div id="sidebar">
```

Comment:
Please also update the highlighted lines

Reply:
Updated